proposals/lip-0014.md

```diff
@@ -471,6 +477,9 @@ Assume the mechanism described in this section is triggered according to the con
 (A.height<B.height && A.heightPrevoted==B.heightPrevoted)
 ```
 
+It is again essential that a node does not forge any blocks while performing the following steps for switching to a different chain. Otherwise, the node may forge a block contradicting the fork choice rule. This means that forging must be deactivated before starting the following steps.
```

Comment:
Is this `forging off` requirement not better in place as soon as Case 5 occurred and moving to a different chain starts? Perhaps it is clearer to set it as part of the 3 steps of that subsection. Also, this way we avoid repeating this requirement twice in this and previous section and maybe just a short reminder here is enough.

Reply:
I initially thought so, too. But then I intentionally put it after all the checks in the section `Moving to a Different Chain` are done so that forging does not stop because a delegate receives a block with incorrect information from a peer (e.g.,  invalid `heightPrevoted`).